server/api/routes/api.js

```diff
@@ -31,7 +31,7 @@ router.get('/seed', async (req, res) => {
 
   const queryParams = new URLSearchParams({
     seed_genres: genre,
-    limit: 10
+    limit: 12
```

Comment:
This addition was because at the end of spotify's autoplay, if there is not a next song, it will create an error in my app.